std/algorithm/package.d

```diff
@@ -17,25 +17,25 @@ $(TR $(TH Category) $(TH Submodule) $(TH Functions)
 $(TR $(TDNW Searching)
      $(TDNW $(SUBMODULE searching))
      $(TD
-        $(SUBREF searching, all) 
```

Comment:
Assuming my sanity is correct, shouldn't these changes normally be in their own commit? (they are just whitespace changes due to there being extra blanks at the end of the line)